src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
I think it's not the best idea to style tags directly and create a separate class instead.